app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -334,7 +196,105 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
       .subscribe(__ -> koala.trackDiscoveryFilters());
   }
 
-  private static boolean isSuccessfulLogin(final @NonNull ActivityResult activityResult) {
-    return activityResult.isOk() && activityResult.isRequestCode(ActivityRequestCodes.LOGIN_FLOW);
+  private final PublishSubject<NavigationDrawerData.Section.Row> childFilterRowClick = PublishSubject.create();
+  private final PublishSubject<Void> internalToolsClick = PublishSubject.create();
+  private final PublishSubject<Void> loggedOutLoginToutClick = PublishSubject.create();
+  private final PublishSubject<InternalBuildEnvelope> newerBuildIsAvailable = PublishSubject.create();
+  private final PublishSubject<Boolean> openDrawer = PublishSubject.create();
+  private final PublishSubject<Integer> pageChanged = PublishSubject.create();
+  private final PublishSubject<Integer> pagerCreatedPage = PublishSubject.create();
+  private final PublishSubject<NavigationDrawerData.Section.Row> parentFilterRowClick = PublishSubject.create();
+  private final PublishSubject<Void> profileClick = PublishSubject.create();
+  private final PublishSubject<Void> settingsClick = PublishSubject.create();
+  private final PublishSubject<NavigationDrawerData.Section.Row> topFilterRowClick = PublishSubject.create();
+
+  private final PublishSubject<List<Integer>> clearPages = PublishSubject.create();
```

Comment:
some of the clarity from cleaning up these VMs is being able to ask ourselves questions like, "Why are some outputs `PublishSubjects`?" 🤔